Library/AppEnvironment.swift

```diff
@@ -295,24 +296,26 @@ public struct AppEnvironment {
                                                    ubiquitousStore: KeyValueStoreType,
                                                    userDefaults: KeyValueStoreType) {
 
-    let data: [String:AnyObject?] = [
-      "apiService.oauthToken.token": env.apiService.oauthToken?.token,
-      "apiService.serverConfig.apiBaseUrl": env.apiService.serverConfig.apiBaseUrl.absoluteString,
-      "apiService.serverConfig.apiClientAuth.clientId": env.apiService.serverConfig.apiClientAuth.clientId,
-      "apiService.serverConfig.basicHTTPAuth.username": env.apiService.serverConfig.basicHTTPAuth?.username,
-      "apiService.serverConfig.basicHTTPAuth.password": env.apiService.serverConfig.basicHTTPAuth?.password,
-      "apiService.serverConfig.webBaseUrl": env.apiService.serverConfig.webBaseUrl.absoluteString,
-      "apiService.language": env.apiService.language,
-      "config": env.config?.encode(),
-      "currentUser": env.currentUser?.encode()
-    ]
-
-    userDefaults.setObject(data.compact(), forKey: environmentStorageKey)
+    var data: [String:Any] = [:]
+
+    data["apiService.oauthToken.token"] = env.apiService.oauthToken?.token
+    data["apiService.serverConfig.apiBaseUrl"] = env.apiService.serverConfig.apiBaseUrl.absoluteString
+    // swiftlint:disable line_length
```

Comment:
oh the 3 lines that follow the disable go over 110, but it's just a simple assignment so thought better to ignore than to break it into another line